recipes/dexp/meta.yaml

```
@@ -0,0 +1,78 @@
+{% set name = "dexp" %}
+{% set version = "2021.12.21.844" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dexp-{{ version }}.tar.gz
+  sha256: 310d63c2509fc907abdb1b29bb90baebffd838c64a9119d4211badc07830a0b6
+
+build:
+  entry_points:
+    - dexp=dexp.cli.dexp_main:cli
+    - video=dexp.cli.video_main:cli
+    - install=dexp.cli.install_main:cli
```

I don't think we should ship these executables with these names. It's a built in command in Linux and macOS, at least. What about prepending `dexp` to both?

```suggestion
    - dexp-video=dexp.cli.video_main:cli
    - dexp-install=dexp.cli.install_main:cli
```

---

@royerloic

---

Pinging @JoOkuma